go/test/endtoend/backup/pitr/backup_mysqlctld_pitr_test.go

```diff
@@ -52,161 +52,181 @@ func waitForReplica(t *testing.T) {
 }
 
 // TestIncrementalBackupMysqlctld - tests incremental backups using myslctld
-func TestIncrementalBackupMysqlctld(t *testing.T) {
```

Comment:
Most of the content here is the same, just indented, and executes for any one of the three backup types we support. Best reviewed [with spaces ignored](https://github.com/vitessio/vitess/pull/13156/files?w=1).